doc/SLSKPROTOCOL.md

```diff
@@ -271,6 +271,7 @@ but it handles the protocol well enough (and can be modified).
 | 151  | [Leave Global Room](#server-code-151)             | Deprecated |
 | 152  | [Global Room Message](#server-code-152)           | Deprecated |
 | 153  | [Related Searches](#server-code-153)              | Obsolete   |
+| 160  | [Excluded Search Phrases](#server-code-160)       |            |
```

Comment:
Excluded -> Disallowed

This is clearer that the phrases are lawfully prohibited.